mapper/javabean/src/main/java/org/hibernate/search/mapper/javabean/model/impl/JavaBeanTypeModel.java

```diff
@@ -138,7 +138,6 @@ public boolean isSubTypeOf(Class<?> superClassCandidate) {
 	@Override
 	public Stream<PojoPropertyModel<?>> getDeclaredProperties() {
 		return Arrays.stream( declaredBeanInfo.getPropertyDescriptors() )
-				.filter( descriptor -> descriptor.getReadMethod() != null )
```

Comment:
Making a test for it. Even with `java.beans.Introspector`. It seems that if there is no getter, there is no PropertyDescriptor for it. So why do we need to filter it?

Reply:
Because that's not true? As I explained above, there can be a setter only.

```
public class MyTest {

	@Test
	public void test() throws IntrospectionException {
		BeanInfo beanInfo = Introspector.getBeanInfo( MyBean.class );
		PropertyDescriptor[] propDesc = beanInfo.getPropertyDescriptors();
		for ( PropertyDescriptor propertyDescriptor : propDesc ) {
			System.out.println( "Property '" + propertyDescriptor.getName() + "', read method: " + propertyDescriptor.getReadMethod() );
		}
	}

	public static class MyBean {

		public void setFoo(int foo) {

		}
	}
}
```

Output:

```
Property 'class', read method: public final native java.lang.Class java.lang.Object.getClass()
Property 'foo', read method: null
```